code42_connector.py

```diff
@@ -491,11 +503,30 @@ def _handle_list_cases(self, param, action_result):
     def _handle_add_case_event(self, param, action_result):
         case_number = param["case_number"]
         event_id = param["event_id"]
-        self._client.cases.file_events.add(case_number=case_number, event_id=event_id)
+        try:
+            self._client.cases.file_events.add(
+                case_number=case_number, event_id=event_id
+            )
+        except Py42BadRequestError as err:
+            if "NO_SUCH_CASE" in err.response.text:
```

Comment:
I wish these were just in py42 instead

Reply:
Yeah, ideally we'd add a `Py42CaseNotFoundError` and maybe a `Py42EventNotFoundError` too, this would clean up the CLI errors for missing cases too.